Kickstarter-iOS/Views/Cells/SettingsAccountPickerCell.swift

```diff
@@ -0,0 +1,75 @@
+import Prelude
+import Library
+import KsApi
+
+internal protocol SettingsAccountPickerCellDelegate: class {
+  /// Called after user selects currency in picker to remove picker cell
+  func shouldDismissCurrencyPicker()
+}
+
+final class SettingsAccountPickerCell: UITableViewCell, NibLoading, ValueCell {
+  internal var delegate: SettingsAccountPickerCellDelegate?
+  @IBOutlet fileprivate weak var pickerView: UIPickerView!
+  @IBOutlet fileprivate var lineLayer: [UIView]!
+
+  private let viewModel: SettingsAccountPickerCellViewModelType = SettingsAccountPickerCellViewModel()
+
+  override func awakeFromNib() {
+    super.awakeFromNib()
+
+    self.pickerView.delegate = self
+    self.pickerView.dataSource = self
+  }
+
+  func configureWith(value cellValue: SettingsCellValue) {
+    self.viewModel.inputs.configure(with: cellValue)
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = lineLayer
+      ||> separatorStyle
+  }
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.notifyCurrencyPickerCellRemoved
+      .observeForUI()
+      .observeValues { [weak self] _ in
+        self?.delegate?.shouldDismissCurrencyPicker()
+    }
+  }
+}
+
+// MARK: UIPickerViewDataSource & UIPickerViewDelegate
+
+extension SettingsAccountPickerCell: UIPickerViewDataSource {
+  func numberOfComponents(in pickerView: UIPickerView) -> Int {
+    return 1
+  }
+
+  func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+    return Currency.allCases.count
+  }
+}
+
+extension SettingsAccountPickerCell: UIPickerViewDelegate {
+  func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+    return Currency(rawValue: row)?.descriptionText
+  }
+
+  func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+    guard let selectedCurrency = Currency(rawValue: row) else {
+      return
+    }
+
+    NotificationCenter.default.post(name: Notification.Name.ksr_updatedCurrencyCellDetailText,
```

Comment:
There might be a way we can avoid using notifications here once we have the currency text being read from the `User` object. Can we add a `//TODO` here to remove this notification once the currency can be read from the user object?